src/libraries/System.Net.Http/src/System/Net/Http/HttpMethod.cs

```diff
@@ -64,9 +64,6 @@ public static HttpMethod Patch
             get { return s_patchMethod; }
         }
 
-        // Don't expose CONNECT as static property, since it's used by the transport to connect to a proxy.
-        // CONNECT is not used by users directly.
-
         public static HttpMethod Connect
```

Comment:
Unrelated, seems like we're missing an XML comment that would feed into getting this documented.  Did it get documented through another means?

Reply:
#74732 lists it as undocumented

Reply:
I will document it in dotnet-api-docs xml directly when dealing with #74732